<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingOrderBrowser</class>
 <widget class="QDialog" name="BookingOrderBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0,0,0">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnNew">
         <property name="text">
          <string>New Booking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel Booking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSkip">
         <property name="text">
          <string>Skipped Booking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPrint">
         <property name="text">
          <string>Print Bookings ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Show Orders:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbDateFilter">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTreeWidget" name="twOrders">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="5">
           <widget class="QPushButton" name="btnSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QWidget" name="widget" native="true"/>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="cbFilter">
            <item>
             <property name="text">
              <string>Client Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Order Title</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Booking Period</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnClearFilter">
            <property name="text">
             <string>Clear Filter</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnSelectFilter">
            <property name="text">
             <string>Select Filter ...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="6">
           <widget class="QStackedWidget" name="swSearch">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="edtFilter"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QComboBox" name="cbBookPeriod">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Last Month</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>This Month</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Next Month</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
